keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+
+
+We will enhance `kepctl` to add an `accept` command that will perform any available validation and handle the movement of the `receipt`:
+
+```
+$ tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    │   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    │   └── 0004.yaml
+    └── removed
+        └── 0003.yaml
+
+$ kepctl accept 0004 --at-risk --release v1.22
+==> KEP 0004 moved to at-risk
+
+$tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    |   └── 0004.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    └── removed
+        └── 0003.yaml
+```
+
+Those receipts marked `at-risk`, will be periodically reviewed and moved to `accepted` once all required elements have been completed. In case, the requirements are not met by Enhancements Freeze, the
+
+
+### Removing a KEP from the Release
+
+Throughout the release cycle, a KEP may be removed from the release. This often occurs after `enhancement freeze`, when an at-risk KEP does not meet the specified stage requirements, or after `code freeze`, if all required implementation is not complete. Once these dates occur, the enhancements team will move the associated receipt to the `removed` directory of the release and open a PR to update the repository.
+
+At this point, the KEP is no longer considered part of the release, and an exception must be granted in order for the KEP to be accepted back into the release.
+
+We will enhance `kepctl` to add a `remove` command that will handle the movement of the `receipt`:
+
+```
+$ tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    |   └── 0004.yaml
+    └── removed
+        └── 0003.yaml
+
+$ kepctl remove 0004 --release v1.22
+==> KEP 0004 moved to removed
+
+$tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    └── removed
+        └── 0003.yaml
+        └── 0004.yaml
+```
+
+
+### Requesting an Exception for a KEP
+
+To request an exception for the release, the responsible SIG will create a new PR that moves the receipt into the `exception` directory. A PR template  in the `kubernetes/enhancement` repo will capture the relevant information for the exception request. The release team will approve the exception reqeust, but will require confirmation from any impacted reviewers on the PR before it is merged.
```

Comment:
This removes the use of a mailing list for exceptions because SIGs will just create a PR, right? Is that going to be a problem? I thought transparency / communication with everyone in regards to exceptions were important?

Reply:
Rereading this and I have the same concern that we are removing the communication of the exception process and also removing the ability to explain why something was accepted and rejected to people who may not watch the repository closely.